lib/web/integrations_awsoidc.go

```diff
@@ -1041,6 +1041,18 @@ func (h *Handler) awsOIDCCreateAWSAppAccess(w http.ResponseWriter, r *http.Reque
 		return nil, trace.Wrap(err)
 	}
 
+	// If the integration name contains a dot, then the proxy must provide a certificate allowing *.<something>.<proxyPublicAddr>
+	if strings.Contains(integrationName, ".") {
+		// Teleport Cloud only provides certificates for *.<tenant>.teleport.sh, so this would generate an invalid address.
+		if h.GetClusterFeatures().Cloud {
+			return nil, trace.BadParameter(`Invalid integration name for enabling AWS Access. Please re-create the integration without the "."`)
```

Comment:
I think it is helpful if a bad parameter error includes the actual bad parameter in the error message. Perhaps this doesn't matter much if there is rarely more than one AWS Access integration though.

Reply:
In this case the only parameter will be the integration name. I'll add it to the error message.